terminus-ui/input/src/input.component.ts

```diff
@@ -843,6 +843,9 @@ export class TsInputComponent implements
   @Output()
   public readonly inputFocus: EventEmitter<boolean> = new EventEmitter();
 
+  @Output()
```

Comment:
Need a comment matching the format of other Outputs.

Also need a test for this. (should be able to find an example by searching for one of the other Outputs in the spec file)